charts/base-cluster/templates/monitoring/kube-prometheus-stack/_config.yaml

```diff
@@ -173,6 +173,7 @@ prometheusOperator:
 defaultRules:
   disabled:
     etcdHighNumberOfFailedGRPCRequests: true
+    etcdDatabaseHighFragmentationRatio: true
```

Comment:
So we disable the etcd alerts and not fix them?

Reply:
Ups, this slid in 🤭